ivar/1.3.2/Dockerfile

```diff
@@ -0,0 +1,101 @@
+FROM ubuntu:jammy as app
+
+# for easy upgrade later. ARG variables only persist during image build
+ARG SAMTOOLSVER=1.16
+ARG HTSLIBVER=1.16
+ARG IVARVER=1.3.2
+
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="iVar"
+LABEL software.version=${IVARVER}
+LABEL description="Computational package that contains functions broadly useful for viral amplicon-based sequencing."
+LABEL website="https://github.com/andersen-lab/ivar"
+LABEL license="https://github.com/andersen-lab/ivar/blob/master/LICENSE"
+LABEL maintainer="Erin Young"
+LABEL maintainer.email="eriny@utah.gov"
+LABEL maintainer2="Curtis Kapsak"
+LABEL maintainer2.email="kapsakcj@gmail.com"
+LABEL modified.from="https://github.com/andersen-lab/ivar/blob/master/Dockerfile"
+LABEL original.maintainer="Kathik G"
+LABEL original.maintainer.email="gkarthik@scripps.edu"
+
+# install dependencies, cleanup apt garbage.
+# bedtools version=2.26.0
+# keeping vim and nano for legacy reasons
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  build-essential \
+  ca-certificates \
+  procps \
+  autoconf \
+  autotools-dev \
+  automake \
+  zlib1g-dev \
+  python3 \
+  wget \
+  libbz2-dev \
+  liblzma-dev \
+  libncurses-dev \
+  git \
+  bedtools \
+  python3-pip \
+  vim \
+  nano && \
+  apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# installing htslib
+RUN wget https://github.com/samtools/htslib/releases/download/${HTSLIBVER}/htslib-${HTSLIBVER}.tar.bz2 && \
+  tar xvf htslib-${HTSLIBVER}.tar.bz2 && \
```

Comment:
I thought you needed `tar -j` for bzip2-compressed files? looks like in uncompressed fine without it 🤷 

Also, the `tar` command is quite verbose - might be good to remove the `v` option from here (and other lines) to shorten the build output